### PR TITLE
Add dasbus support to anaconda_updates

### PR DESCRIPTION
**Pull request by jkonecny12**
Dasbus is our new library to handle DBus communication.

Add ``--dasbus`` parameter to the program to copy library in the updates image more easily.

### CODE REVIEW

File: anaconda_updates/update_image.py [hunk 1]
```diff
@@ -187,6 +193,15 @@ def prepare(self):
...
                 print("Skipping simpleline copy:", str(e))
 
+        if GlobalSettings.use_dasbus:
+            print("Copy dasbus...")
+            source = os.path.join(GlobalSettings.projects_path, "dasbus/dasbus")
+            dest = os.path.join(GlobalSettings.projects_path, GlobalSettings.anaconda_path, "updates/run/install/updates/dasbus")
+            try:
+                shutil.copytree(source, dest)
+            except FileExistsError as e:
+                print("Skipping simpleline copy:", str(e))
```
Comment (poncovka) [inline anaconda_updates/update_image.py]: Skipping dasbus copy ...

**Review by poncovka — APPROVED** [review]
Otherwise, it looks good to me. Thank you!

File: anaconda_updates/update_image.py (hunk 1 above)
Comment (jkonecny12) [reply anaconda_updates/update_image.py]: ups, fixed

**Comment by jkonecny12** [comment]
**UPDATED:**
* Thanks @poncovka for fixing that leftover!

**Review by poncovka — APPROVED** [review]
Looks good to me now.

**Pull request merged**